lib/Graph/Graph.cpp

```diff
@@ -1066,6 +1066,35 @@ LogNode *Function::createLog(llvm::StringRef name, NodeValue input,
   return addNode(new LogNode(name, outTy ? outTy : input.getType(), input));
 }
 
+Node *Function::createLogit(llvm::StringRef name, NodeValue data, float eps) {
```

Comment:
`NodeValue data` --> `NodeValue input`

so that the parameter name here matches the parameter name in the header.

Reply:
Thanks, done!